card/src/main/java/com/adyen/checkout/card/internal/ui/view/SecurityCodeField.kt

```diff
@@ -34,7 +34,13 @@ internal fun SecurityCodeField(
     val supportingTextSecurityCode = if (showSecurityCodeError) {
         securityCodeState.errorMessage?.let { resolveString(it) }
     } else {
-        null
+        resolveString(
+            if (isAmex == false) {
+                CheckoutLocalizationKey.CARD_SECURITY_CODE_HINT_4_DIGITS
+            } else {
+                CheckoutLocalizationKey.CARD_SECURITY_CODE_HINT_3_DIGITS
+            }
```

Comment:
Here the following is gonna happen:
- The card number field is empty: Text for 3 digits
- The card brand is Amex: Text for 3 digits
- The card brand is not Amex: Text for 4 digits.

But the correct one should be:
- The card number field is empty: Text for 3 digits
- The card brand is Amex: Text for 4 digits
- The card brand is not Amex: Text for 3 digits.

To reflect the conversation in the previous PR, this could be adjusted:
```suggestion
            if (sAmex == true) {
                CheckoutLocalizationKey.CARD_SECURITY_CODE_HINT_4_DIGITS
            } else {
                CheckoutLocalizationKey.CARD_SECURITY_CODE_HINT_3_DIGITS
            }
```